components/Carousel/Carousel.jsx

```diff
@@ -46,7 +46,7 @@ class Carousel extends Component {
       width: 0, // passed down to <Slide> so it can reuse the h/w calculations
     };
     this.el = null;
-    this.autoplayInterval = null;
+    // this.autoplayInterval = null;
```

Comment:
Let's just set `this.timerID = null` in the constructor instead of this

Reply:
done!